scripts/gulpfile.factory.js

```diff
@@ -150,7 +123,7 @@ module.exports = function factory({
   };
 
   task("merge", () => {
-    return src(`${elementName}.js`, {
+    return src(`${elementName}*.js`, {
```

Comment:
I would like to understand the need for `*` here.  Previously, only the "main" js module (ex: `pfe-avatar.js`) will get merged, meaning the HTML and CSS will get embedded into it.  Utility modules (ex: `hslrgb.js`) don't, because they don't contain web component definitions.

Is this to enable support for splitting up big files like `pfe-accordion.js` into `pfe-accordion-header.js`, etc?  That would be awesome.  I glanced through all the code for the merge task and it seems like it should work.

Reply:
Exactly, it enables separate JS files for a component that have their own html and scss/css files.  This will require that any such files that are dependencies of the parent must at least start with the name of the component.  So `pfe-accordion-header.js` instead of something like `header.js`.

Reply:
Cool, that sounds great to me!